README.md

```diff
@@ -1,183 +1,139 @@
 # llnode
 
-[![Build Status](https://travis-ci.org/nodejs/llnode.svg?branch=master)](https://travis-ci.org/nodejs/llnode)
+[![npm](https://img.shields.io/npm/v/node-core-utils.svg?style=flat-square)](https://npmjs.org/package/node-core-utils)
```

Comment:
This links to `node-core-utils`

Reply:
@cjihrig Ooops, copy-paste error...fixed, thanks!